src/main/target/CC3D/target.h

```diff
@@ -72,8 +72,13 @@
 #define USE_VCP
 #define USE_UART1
 #define USE_UART3
+
+#ifdef CC3D_OPBL
```

Comment:
We do have a general problem here. All target config files have their own unique layout, it is hard to compare two different target configs. A strict format would be good, a template. Or a config tool generating target files. The later a bit of overkill I would say...